stateful-workload-filestore/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:latest
+RUN mkdir htmlTemp
+RUN cd htmlTemp
+WORKDIR /htmlTemp
+ADD indexInfo.html /htmlTemp/
```

Comment:
`ADD` already creates a directory if it doesn't already exist, and `RUN cd` doesn't actually do anything in this context.

You could simplify like this:
```
FROM alpine:latest
WORKDIR /htmlTemp
ADD indexInfo.html /htmlTemp/
```

Also, should there be a `CMD` or `ENTRYPOINT`? What is this image running exactly?